### PR TITLE
Add more BCN 2024 info

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
@netlify /events/2024/hackathon-barcelona

### CODE REVIEW

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 1]
```diff
@@ -17,50 +17,145 @@ locations:
...
-    <span>
-        <i class="fa-solid fa-map-pin me-2"></i>
-        <span>Google Maps</span>
-    </span>
-</a>
+Beginners welcome, but please note that this is not a training event. There will be a separate [training event](https://summit.nextflow.io/2024/barcelona/training/) in parallel to the hackathon. We will all meet up for the social evening.
+
+## How to connect with others
+
+We use slack for all communication throughout the year. Joining ahead of the hackathon gives you an opportunity to already meet
```
Comment (mashehu) [inline sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:69]:
```suggestion
We use Slack for all communication throughout the year. Joining ahead of the hackathon gives you an opportunity to already meet
```

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Comment by ewels** [comment]
Did a bit more organising / wrote a bit more. Also added some placeholder projects to give some structure.

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 2]
```diff
@@ -13,54 +13,143 @@ locations:
...
+The nf-core hackathons are collaborative, community-driven events where participants work together on projects.
 
-<a class="btn btn-success btn-lg mb-3" href="https://summit.nextflow.io/2024/barcelona/register/">
-    <span>
-        <i class="fa-solid fa-pencil me-2"></i>
-        <span>Register now</span>
-    </span>
-</a>
+Everyone is welcome, no prior experience of nf-core contributions is needed. However, we do expect that you have some experience with writing Nextflow code.
+Beginners are welcome, but this is not a training event.
```
[the remark this answers is not in the document]
Comment (ewels) [reply sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:42]:
```suggestion
```

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 3]
```diff
@@ -13,54 +13,143 @@ locations:
...
+_(There is a separate [training event](https://summit.nextflow.io/2024/barcelona/training/) in parallel to the hackathon.)_
```
Comment (ewels) [inline sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:43]:
```suggestion
Note that there is a separate [training event](https://summit.nextflow.io/2024/barcelona/training/) for learning Nextflow and nf-core from scratch, running in parallel to the hackathon.
```

**Review by ewels — APPROVED** [review]
I think that this is good to go. Definitely room for more content and improvement, but it's better than what we have currently so keen to merge ASAP. Thanks both!

**Pull request merged**